workspaces/ui-v2/config/modules.js

```diff
@@ -65,28 +65,7 @@ function getWebpackAliases(options = {}) {
 
   if (path.relative(paths.appPath, baseUrlResolved) === '') {
     return {
-      src: paths.appSrc,
-    };
-  }
-}
-
-/**
- * Get jest aliases based on the baseUrl of a compilerOptions object.
- *
- * @param {*} options
- */
-function getJestAliases(options = {}) {
```

Comment:
So this isn't currently used, and I don't believe jest actually runs through webpack (if it does, it'd use the same config). Jest runs off node and i _think_ it uses it's own module dependency resolution (which I specified in `jest.config.js`)

Deleting this to try to avoid confusion (webpack config is already confusing :/)